openeo/__init__.py

```diff
@@ -16,7 +16,7 @@ class BaseOpenEoException(Exception):
 from openeo.rest.datacube import DataCube, UDF
 from openeo.rest.connection import connect, session, Connection
 from openeo.rest.job import BatchJob, RESTJob
-
```

Comment:
I think we should only expose `LocalConnection`  here once this feature has some maturity, otherwise we risk  to "vendor-lock-in" ourselves  in a corner